htlcswitch/link.go

```diff
@@ -1531,8 +1531,11 @@ func (l *channelLink) processHtlcResolution(resolution invoices.HtlcResolution,
 		// result.
 		failure := getResolutionFailure(res, htlc.pd.Amount)
 
+		//nolint:forcetypeassert
```

Comment:
what's the reason for skipping this lint?

Reply:
This gets churned in later commits. I didn't want to go through and add a whole bunch of ceremony for something that was immediately going away. If you look at the final diff you won't see this appear.